libvast/vast/fwd.hpp

```diff
@@ -200,6 +203,20 @@ struct v0;
 
 } // namespace fbs
 
+namespace detail {
+
+struct stable_map_policy;
+
+template <class, class, class, class>
+class vector_map;
+
+/// A map abstraction over an unsorted `std::vector`.
```

Comment:
```suggestion
```

Reply:
This is an alias definition with a defaulted template argument and we can only have it once, so I moved it over with its documentation.